coreneuron/nrniv/main1.cpp

```diff
@@ -368,15 +372,27 @@ const char* nrn_version(int) {
 
 using namespace coreneuron;
 
-extern "C" void mk_mech_init(int argc, char** argv) {
-#if NRNMPI
-    nrnmpi_init(1, &argc, &argv);
-#endif
+
+extern "C" int mk_mech_init(int argc, char** argv) {
```

Comment:
Small detail.. I did not understand why `mk_mech_init()` should be `int` since it doesn't return any value

Reply:
For whatever reason, if I let it `void`, the compilation would just fail. I'll check it out again.